tutorials/helper_functions.py

```diff
@@ -8,6 +8,7 @@
 import requests
 
 import mano
+from mano import sync as msync
```

Comment:
I see, then let's follow the `mano` documentation for consistency:
`import mano.sync as msync`
https://github.com/harvard-nrg/mano#api-for-downloading-data